lib/Relation.php

```diff
@@ -0,0 +1,262 @@
+<?php
+/**
+ * Interface
+ *
+ * @package ActiveRecord
+ */
+
+namespace ActiveRecord;
+
+use ActiveRecord\Exception\RecordNotFound;
+
+class Relation
+{
+    /**
+     * @var array<string>
+     */
+    private array $alias_attribute;
+    private Table $table;
+
+    /**
+     * @var array<string, mixed>
+     */
+    private array $options = [];
+
+    /**
+     * @param class-string  $className
+     * @param array<string> $alias_attribute
+     */
+    public function __construct(string $className, array $alias_attribute)
+    {
+        $this->alias_attribute = $alias_attribute;
+        $this->table = Table::load($className);
```

Comment:
Should think about guarding this in a getter, so no queries happen until we're sure we need it.

Reply:
Good idea. Implemented.